NorthstarDLL/squirrel/squirrel.cpp

```diff
@@ -256,7 +257,16 @@ template <ScriptContext context> void SquirrelManager<context>::VMCreated(CSquir
 
 		defconst(m_pSQVM, pair.first.c_str(), bWasFound);
 	}
+
```

Comment:
no, this was not an accidental change. I am splitting up the const definitions for readability